models/common/fhir-path-builder.ts

```diff
@@ -0,0 +1,56 @@
+/*
+* Fhir Path Builder, used to extract values from FHIR.
+* see http://hl7.org/fhirpath/ for the specification
+* and https://github.com/HL7/fhirpath.js#readme
+* for the latest implementation
+*/
+
+export class FhirPathBuilder {
+  bundle(): BundlePathBuilder {
+    return new BundlePathBuilder("Bundle.entry.resource")
```

Comment:
i think it makes sense if it was possible to access fields on the Bundle from BundlePathBuilder, for example `Bundle.total`. should we move the `.entry.resource` bit inside the BundlePathBuilder where it is used?

Reply:
As mentioned, this would be a nice to have, if more classes will be added to handle additional FHIR resources